app/models/ApplicationData.scala

```diff
@@ -36,17 +36,14 @@ object ApplicationData {
   object ApplicationStatus extends Enumeration {
     type ApplicationStatus = Value
     // format: OFF
-    val REGISTERED, CREATED, IN_PROGRESS, SUBMITTED, WITHDRAWN, ONLINE_TEST_INVITED,
-    ONLINE_TEST_STARTED, ONLINE_TEST_COMPLETED, ONLINE_TEST_EXPIRED,
-    ALLOCATION_CONFIRMED, ALLOCATION_UNCONFIRMED, AWAITING_ALLOCATION,
-    ONLINE_TEST_FAILED, ONLINE_TEST_FAILED_NOTIFIED, AWAITING_ONLINE_TEST_RE_EVALUATION, FAILED_TO_ATTEND,
+    val REGISTERED, CREATED, IN_PROGRESS, SUBMITTED, WITHDRAWN, PHASE1_TESTS,
```

Comment:
Aren't we going to have a separate application status for the different steps inside "Phase1 tests" like we had before in Fast Track?